app/views/user_mailer/admin_hidden_bookmark_notification.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for :message do %>
+  <p><%= t("mailer.general.greeting.formal_html", name: style_bold(@user.login)) %></p>
+
+  <p><%= t(".hidden.html", bookmark_url: style_creation_link("bookmark", @bookmark)) %></p>
```

Comment:
The link text should be "bookmark of %{title of bookmarkable item}" and should be translatable.

Also, the variable here should be `bookmark_link` instead of `bookmark_url`.

Note also from Jira: In the HTML email, the creation title inside the bookmark link should be bold and italicized.